### PR TITLE
feat(ecs): ability to access tag parameter value of TagParameterContainerImage

### PR DESCRIPTION
**Pull request by alastair-watts-avrios**
Allows you to use the tag elsewhere within the container definition (e.g. to inform monitoring services of the release version).

Fixes: #13202

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Review by skinny85 — APPROVED** [review]
Looks great @alastair-watts-avrios , thanks so much for the contribution!

Did you mark your PR as "Allows edits from contributors"? It's needed so that the Mergify bot can update your PR with master, and finally merge it in.

File: packages/@aws-cdk/aws-ecs/test/images/tag-parameter-container-image.test.ts [hunk 1]
```diff
@@ -19,6 +19,21 @@ nodeunitShim({
         SynthUtils.synthesize(stack);
       }, /TagParameterContainerImage must be used in a container definition when using tagParameterName/);
 
+      test.done();
+    },
```
Comment (skinny85) [inline packages/@aws-cdk/aws-ecs/test/images/tag-parameter-container-image.test.ts:23]: Can you add an empty line after this line?

```suggestion
    },

```

**Comment by alastair-watts-avrios** [comment]
@skinny85
Have made the requested changes. And managed to set the allow changes option.

**Review by skinny85 — APPROVED** [review]
Thanks @alastair-watts-avrios !

**Pull request merged**